src/components/atoms/Loader.tsx

```diff
@@ -0,0 +1,59 @@
+import { Center } from 'native-base'
+import React, { useMemo } from 'react'
+import { ActivityIndicator, ActivityIndicatorProps } from 'react-native'
+
+import { BricksLoader, BricksLoaderType } from './BricksLoader'
+import { BubblesLoader, BubblesLoaderType } from './BubblesLoader'
+import { CircleLoader, CircleLoaderType } from './CircleLoader'
+import { DiskLoader, DiskLoaderType } from './DiskLoader'
+
+type LoaderType =
+  | {
+      type: 'circle'
+      isFlipped?: boolean
+      color?: string
+      size?: number
+      thickness?: number
+    }
+  | {
+      type: 'bubbles'
+      color?: string
+      size?: number
+    }
+  | {
+      type: 'bricks'
+      size?: number
+      colors?: [string, string, string]
+      containerSize?: number
+      color?: string
+    }
+  | {
+      type: 'disk'
+      circleColor?: string
+      containerColor?: string
+      size?: number
+    }
+  | {
+      type: 'default'
+      color?: string
+      size?: string | number
+    }
+
+export const Loader = ({ type, ...props }: LoaderType): JSX.Element => {
+  const renderLoader = useMemo((): JSX.Element => {
+    if (type === 'circle') {
+      return <CircleLoader {...(props as CircleLoaderType)} />
```

Comment:
Maybe we should unify somehow the props of all Loaders?

Reply:
Do you have any idea?